docs/devicemanager.apib

```diff
@@ -854,3 +854,42 @@ Replaces all attributes from a specific device.
 + Response 500 (application/json)
 
             {"status" : "failed to update device configuration"}
+
+
+### Configure device [PUT /device/{deviceid}/actuate]
+Send a configuration message to the device to change some of its attributes.
+The target attribute must be of type "actuator".
++ Parameters
+ + deviceid: efac (required, string): Device ID being invoked.
+ 
++ Request (application/json)
+    + Headers
+
+            Authorization: Bearer JWT
+
+    + Body
+
+            {
+                "topic": "/admin/efac/config",
```

Comment:
It seems as if this will be ignored (both here, and by current iotagent implementation).
Perhaps it'd be best if removed from docs?

Reply:
Sure! I'll re-re-recheck the documentation.